### PR TITLE
[15.0][ADD] purchase_line_reassign: New module

### PR DESCRIPTION
**Pull request by sergio-teruel**
ping @Tecnativa TT37719

ping @carlosdauden @CarlosRoca13

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

File: purchase_line_reassign/wizards/purchase_line_reassign_wiz.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    )
+    partner_id = fields.Many2one(
+        comodel_name="res.partner",
+        string="Vendor",
+        default=lambda self: self._default_partner_id(),
+    )
+
+    def action_apply(self):
+        active_ids = self.env.context.get("active_ids")
+        lines = self.env["purchase.order.line"].browse(active_ids)
```
Comment (mmequignon) [inline purchase_line_reassign/wizards/purchase_line_reassign_wiz.py:36]: Why browse active ids twice, instead of adding a line_ids field on the wizard?

Comment (sergio-teruel) [reply purchase_line_reassign/wizards/purchase_line_reassign_wiz.py:36]: If I add a field means than I have do create operations in database. I do not need this due to performance

**Review by cesar-tecnativa — APPROVED** [review]

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
The SVG icon doesn't correspond. Remove it

**Comment by sergio-teruel** [comment]
@pedrobaeza changes done

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**